variables.md

```diff
@@ -66,8 +66,8 @@ EXTRABOOTPARAMS_BOOT_LOCAL | string | | Boot options applied during the boot pro
 EXTRABOOTPARAMS_DELETE_CHARACTERS | string | | Characters to delete from boot prompt.
 EXTRABOOTPARAMS_DELETE_NEEDLE_TARGET | string | | If specified, go back with the cursor until this needle is matched to delete characters from there. Needs EXTRABOOTPARAMS_BOOT_LOCAL and should be combined with EXTRABOOTPARAMS_DELETE_CHARACTERS.
 EXTRATEST | boolean | false | Enables execution of extra tests, see `load_extra_tests`
+FIRST_BOOT_CONFIG | string | combustion+ignition | The method used for initial configuration of MicroOS images (combustion/ignition/wizard). Needs to have a matching HDD attached.
```

Comment:
```suggestion
FIRST_BOOT_CONFIG | string | combustion+ignition | The method used for initial configuration of MicroOS images. Possible values are: combustion | ignition | combustion+ignition | wizard`. For ignition/combustion, the job needs to have a matching HDD attached.
```

Reply:
and I know I suggested it, but what about `FIRST_BOOT_MODE` ?

Reply:
```suggestion
FIRST_BOOT_CONFIG | string | combustion+ignition | The method used for initial configuration of MicroOS images. Possible values are: "combustion" "ignition" "combustion+ignition" "wizard". For ignition/combustion, the job needs to have a matching HDD attached.
```

Reply:
Changed